docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+* A Liberty file of the standard cell library with PVT characterization, input and output characteristics, timing and power definitions for each cell.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lib/NangateOpenCellLibrary_typical.lib `as an example liberty file.
+
+Adding a new platform additionally requires the following: 
+
+* A validated installation of the OpenROAD flow scripts is available. See instructions [here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html#building-and-installing-the-software).:
+* A general knowledge of VLSI design and RTL to GDS flows.  \
+OpenROAD implements a fully-automated RTL-GDSII but it requires familiarity with the OpenROAD flow scripts flow and the scripts provided to enable debugging in case of problems.Add a flow section briefly describing the OR or ORFS flow steps to add the platform a platform here
+
+
+# Adding a New Platform to OpenROAD <a name="ANPOpenROAD"></a>
+
+
+## Setup <a name="Setup"></a>
+
+This section describes the necessary files and  directories needed to build the platform.  All files and directories made/edited are independent of each other unless otherwise stated.
+
+
+## Makefile <a name="Makefile"></a>
+
+Make the following edits  to the Makefile (located in `OpenROAD-flow-scripts/flow/Makefile`) so that OpenROAD can run the flow on a design using the new platform.
+
+1. At  the beginning of the Makefile, there is a block of `DESIGN_CONFIG` variables that are commented out. These variables tell OpenROAD which design to run and on what platform. `DESIGN_CONFIG` specifically points to a `config.mk` file located in the designs directory for the respective platform. A `DESIGN_CONFIG` variable must be added for the new platform using a specific design. OpenROAD has multiple verilog designs already made which can be used with any platform (see `OpenROAD-flow-scripts/flow/designs/src` for a list of usable designs). For example, a `DESIGN_CONFIG` variable using the `riscv32i` design on a new platform would look as follows:
+
+
+```
+DESIGN_CONFIG=./designs/MyNewPlatform/riscv32i/config.mk
+```
+
+
+The `config.mk` file will be generated later in the [Design Directory](#DesignDirectory) section of this document.
+
+
+## Platform Directory <a name="PlatformDirectory"></a>
+
+Create a directory for the new technology inside `OpenROAD-flow-scripts/flow/platforms` to contain the necessary files for the OpenROAD flow.
+
+
+```
+$ mkdir OpenROAD-flow-scripts/flow/platforms/MyNewPlatform
+```
+
+
+
+## Design Directory <a name="DesignDirectory"></a>
+
+The design directory contains the configuration files for all the designs of a specific platform. Create a directory for the new platform in OpenROAD-flow-scripts/flow/designs to contain the relevant files and directories for all the designs for the flow in that specific platform. Each design requires it’s own config.mk and constraints.sdc files. 
```

Comment:
Backticks around `config.mk` and `constraints.sdc`. Please review other instances in the document, paths and file names should all have it.